### ISSUE TITLE
Kies codestyle en valideer in CI

### ISSUE THREAD

**Issue by ferrydeboer** [codefornl/helpradar#48]
Om consistent leesbare code te krijgen
Als developer
Wil dat we één codestyle & naming convention gebruiken en die in de CI pipeline wordt gevalideerd zodat dit in PR's als check is opgenomen.

**Comment by milovanderlinden** [comment]
Is het gebruiken van de IDE default niet goed? Ik gebruik pycharm en die is redelijk duidelijk over wat ie wel en niet goedkeurt qua coding style

**Comment by ferrydeboer** [comment]
Ja, pyCharm doet al aardig wat inderdaad. Maar dat geld a. niet voor alle IDE's en b. vereist alsnog wel dat je dat dan actief gebruikt. Een linter inzetten maakt het in ieder geval zichtbaar waar er wordt afgeweken zodat je dit in een PR direct kunt rechtzetten.

Het lijkt me ook gewoon het handigst om https://www.python.org/dev/peps/pep-0008/ te gebruiken. Dat is de default ook in PyCharm afaik, maar is wel minder duidelijk volgens mij op bijvoorbeeld method names.
